frontend/src/components/Form.js

```diff
@@ -63,6 +63,7 @@ const Form = ({ setNumberOfErrors, setErrorsExistCounter }) => {
       }
     }
     formData.append('tax_year', 2021);
+    formData.append('status', 'submitted');
```

Comment:
These two attributes are generated in the serializer, I wonder if using a `SerializerMethodField` from DRF would allow us to not post data for those attributes here on the form since this data is not used for creating the model.

Reply:
I tried a SerializerMethodField, but that didn't work. I think that's only used for serializing (going from model to parsed data), but here we're deserializing (going from form data to model).

I also tried making the status a custom field (https://www.django-rest-framework.org/api-guide/fields/#custom-fields) and overriding the to_internal_value() method there in hopes that not posting the status would not trigger a validation error, but no luck there.

In the end, what did work was simply excluding those fields in the serializer (exclude = ["user", "status", "tax_year"]), so I guess I'll just do that?